org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/quickfix/ConstructorsQuickFixTest.xtend

```diff
@@ -379,4 +379,403 @@ class ConstructorsQuickFixTest extends AbstractWollokQuickFixTestCase {
 		assertQuickfix(initial, result, Messages.WollokDslQuickFixProvider_remove_constructor_name, 2)
 	}
 
+	@Test
+	def addNewConstructorInRefClassNoParams(){
+		val initial = #[
+		'''
+		class A {
+			const y
+			constructor(_y){
+				y = _y
+			}
+		
+			method someMethod(){
+				return null
+			}
+		}
+		
+		class B {
+			method b() {
+				const a = new A()
+				console.println(a.toString())
+			}
+		}
+		''']
+
+		val result = #[
+		'''
+		class A {
+			const y
+			constructor(_y){
+				y = _y
+			}
+			constructor() {
+				//TODO: Autogenerated Code ! 
+			}
+		
+			method someMethod(){
```

Comment:
This code seems superfluous, is it?

Reply:
I'm checking how constructor is inserted while having no methods, at least one method, with or without variables, with or without constructors, etc.